sources/Valkey.Glide/GlideClient.cs

```diff
@@ -102,4 +102,22 @@ public async Task<bool> KeyCopyAsync(ValkeyKey sourceKey, ValkeyKey destinationK
         Utils.Requires<NotImplementedException>(flags == CommandFlags.None, "Command flags are not supported by GLIDE");
         return await Command(Request.KeyCopyAsync(sourceKey, destinationKey, destinationDatabase, replace));
     }
+
+    public async Task<ValkeyValue> ClientGetNameAsync(CommandFlags flags = CommandFlags.None)
+    {
+        Utils.Requires<NotImplementedException>(flags == CommandFlags.None, "Command flags are not supported by GLIDE");
+        return await Command(Request.ClientGetName());
+    }
+
+    public async Task<long> ClientIdAsync(CommandFlags flags = CommandFlags.None)
+    {
+        Utils.Requires<NotImplementedException>(flags == CommandFlags.None, "Command flags are not supported by GLIDE");
+        return await Command(Request.ClientId());
+    }
+
+    public async Task<string> SelectAsync(long index, CommandFlags flags = CommandFlags.None)
```

Comment:
Since valkey 9 cluster supports multiDB and select command. We should move this to base client or duplicate for cluster client.